libr/debug/debug.c

```diff
@@ -343,6 +343,11 @@ R_API int r_debug_select(RDebug *dbg, int pid, int tid) {
 	if (dbg->h && dbg->h->select && !dbg->h->select (pid, tid))
 		return false;
 
+	{
+		char buf[32];
+		sprintf (buf, "pid %d", pid);
+		r_io_system (dbg->iob.io, buf);
```

Comment:
Okay, I'm fine with using sdb_fmt. I thought it did a dynamic allocation like r_new_strf